client/client.go

```diff
@@ -58,6 +60,14 @@ func Configure(logger hclog.Logger, config interface{}) (schema.ClientMeta, erro
 	serviceAccountKeyJSON := []byte(providerConfig.ServiceAccountKeyJSON)
 	if len(serviceAccountKeyJSON) == 0 {
 		serviceAccountKeyJSON = []byte(os.Getenv(serviceAccountEnvKey))
+		var v map[interface{}]interface{}
```

Comment:
I am getting an error when unmarshalling with `map[interface{}]interface{}` but it works with `map[string]interface{}`